package-lock.json

```diff
@@ -2115,7 +2115,7 @@
     },
     "array-equal": {
       "version": "1.0.0",
-      "resolved": "https://registry.npmjs.org/array-equal/-/array-equal-1.0.0.tgz",
+      "resolved": "http://registry.npmjs.org/array-equal/-/array-equal-1.0.0.tgz",
```

Comment:
That has nothing to do with the story and looks a bit dangerous - please remove.